### ISSUE TITLE
Is this a revamp of airline booking for Quaser v2

### ISSUE THREAD

**Issue by talhatec** [heitorlessa/serverless-airline-quasar-1x#1]
I picked up the airline booking on Twitch/Youtube(2019) to learn Amplify but sadly nothing works anymore as both Amplify and Quaser have evolved radically.
Is this repo a revamp based on latest versions of Amplify and Quaser (without Vue/Quaser plugin)?

**Comment by heitorlessa** [comment]
Yes! I’ve already upgraded both Quasar and Amplify, and am fixing the last
breaking changes of both frameworks this month.

I’ll merge this to the main repo once I’m done, and create a public
hardcore workshop on this
